db/seeds.rb

```diff
@@ -1,7 +1,128 @@
+maria = User.create(username: 'Maria', email: 'maria@mail.com', password: 'm!a')
+
+# Resolved bets
+bet1 = Bet.create(bet_description: 'Ninjas vs Pirates',
+                  status: 0,
+                  creator_id: bob.id,
+                  expires_at: DateTime.tomorrow)
+b1_option1 = BetOption.create(option_text: 'Ninjas win',
+                              bet: bet1)
+b1_option2 = BetOption.create(option_text: 'Pirates win',
+                              bet: bet1)
+
+UserBet.create(user: bob,
+               bet: bet1,
+               bet_option: b1_option1,
+               amount_bet: 100)
+UserBet.create(user: sarah,
+               bet: bet1,
+               bet_option: b1_option2,
+               amount_bet: 215)
+UserBet.create(user: ivan,
+               bet: bet1,
+               bet_option: b1_option2,
+               amount_bet: 19.99)
+UserBet.create(user: maria,
+               bet: bet1,
+               bet_option: b1_option1,
+               amount_bet: 143.53)
+bet1.resolve(winning_option: b1_option2)
+
+bet2 = Bet.create(bet_description: 'Soccer: England VS Spain',
+                  creator_id: ivan.id,
+                  expires_at: DateTime.tomorrow,
+                  status: 0)
+b2_option1 = BetOption.create(option_text: 'England wins 2:0',
+                              bet: bet2)
+b2_option2 = BetOption.create(option_text: 'Spain wins 4:3',
+                              bet: bet2)
+b2_option3 = BetOption.create(option_text: 'England wins 1:0',
+                              bet: bet2)
+b2_option4 = BetOption.create(option_text: 'Draw',
+                              bet: bet2)
+
+UserBet.create(user: bob,
+               bet: bet2,
+               bet_option: b2_option1,
+               amount_bet: 25.34)
+UserBet.create(user: sarah,
+               bet: bet2,
+               bet_option: b2_option2,
+               amount_bet: 64.24)
+UserBet.create(user: ivan,
+               bet: bet2,
+               bet_option: b2_option3,
+               amount_bet: 34.35)
+UserBet.create(user: maria,
+               bet: bet2,
+               bet_option: b2_option4,
+               amount_bet: 56.99)
+bet2.resolve(winning_option: b2_option4)
+
+# Unresolved bets
+bet3 = Bet.create(bet_description: 'I bet Ivan will not deliver '\
+                  'the project on time',
+                  status: 1,
+                  expires_at: DateTime.tomorrow,
+                  creator_id: sarah.id)
+b3_option1 = BetOption.create(option_text: 'He will',
+                              bet: bet3)
+b3_option2 = BetOption.create(option_text: 'He won\'t',
+                              bet: bet3)
+UserBet.create(bet: bet3,
+               user: bob,
+               bet_option: b3_option2,
+               amount_bet: 13.5)
+UserBet.create(bet: bet3,
+               user: maria,
+               bet_option: b3_option1,
+               amount_bet: 20.99)
+
+# Expired bets
+bet4 = Bet.new
+Timecop.freeze(Date.new(2016, 8, 17)) do
```

Comment:
I want to create an expired bed for testing purposes, with the way the validator is set up right now, it won't allow me to create bet with an already expired date, so I had to do this, hope it makes it more clear.